docs/notebooks/README.md

```diff
@@ -0,0 +1,13 @@
+To launch analyses.ipynb - [![Binder](https://img.shields.io/badge/launch-analyses-blue
+)](https://mybinder.org/v2/gh/HarshithaPrabhuswamy/conflowgen/main?labpath=docs%2Fnotebooks%2Fanalyses.ipynb)
+
+To launch first_steps-ipynb - [![Binder](https://img.shields.io/badge/launch-first_steps-orange)](https://mybinder.org/v2/gh/HarshithaPrabhuswamy/conflowgen/main?labpath=docs%2Fnotebooks%2Ffirst_steps.ipynb)
+
+To launch in_spotlight.ipynb - [![Binder](https://img.shields.io/badge/launch-in_spotlight-purple
+)](https://mybinder.org/v2/gh/HarshithaPrabhuswamy/conflowgen/main?labpath=docs%2Fnotebooks%2Fin_spotlight.ipynb)
+
+To launch input_distributions.ipynb - [![Binder](https://img.shields.io/badge/launch-input_distributions-violet
+)](https://mybinder.org/v2/gh/HarshithaPrabhuswamy/conflowgen/main?labpath=docs%2Fnotebooks%2Finput_distributions.ipynb)
+
+To launch previews.ipynb - [![Binder](https://img.shields.io/badge/launch-previews-green
+)](https://mybinder.org/v2/gh/HarshithaPrabhuswamy/conflowgen/main?labpath=docs%2Fnotebooks%2Fpreviews.ipynb)
```

Comment:
please integrate these links into the respective Jupyter notebooks. I don't want this to be a separate file.

When people visit https://conflowgen.readthedocs.io/en/latest/notebooks/first_steps.html, I want them to see the button to try out first_steps.ipynb. For the other notebooks, I expect to happen the same.